### DISCUSSION TITLE
[Feature] add selection to multiple albums

### DISCUSSION THREAD

**Discussion by enboig** [immich-app/immich#2686] [Feature Request]
### The feature

Some times I would like to some photos to multiple albums. After selecting them and adding to an album, selection is lost so I cannot:
- Add them to another album
- Archive them

One option could be keep the selection after adding them to an album.

The other could be a 5 second bubble allowing to "reselect" them (I don't know how this would look on a tablet or web)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by tbleiker** [comment]
Would love to see this as well.

**Comment by Sherlock1979** [comment]
It would save me a massive amount of time and pain. I need to put dozens of assets to 3-5 albums every day.

I'd really appreciate a fix / enhancement for this one. Thanks.

BTW, the selection is retained on the web but it is lost in the app.

**Comment by WoOS2** [comment]
Please remove the "Web" checkmark as the web interface already retains selection after album assignment (which I had to find out the hard way once ;-)

**Comment by timonrieger** [comment]
for comparison, Apple photos does not keep the selection on mobile either.
I agree that it would be useful tho. The current behavior in Immich is to either…
a) …navigate to the newly created album
b) …add to album and unselect assets (just as Apple does it)

For b) it's straightforward as we would simply NOT unselect the assets. For a) we either don't redirect to the new album and apply the same (preferred) OR keep behavior as is, since arguable one could select all assets in that album again to readd them. The latter would only work if there was a "Select all" button in albums tho, as of now one would have to reselect all.

**Comment by VitasLoWang** [comment]
I have submitted similar request to keep selection after adding TAGS and description. It behaves the same on web and mobile.
https://github.com/immich-app/immich/discussions/31179
